server/controllers/middlewares.js

```diff
@@ -77,20 +79,20 @@ module.exports = function(app) {
         return next();
       }
 
-      var name, email, paypalEmail;
+      let name, email, paypalEmail;
 
       // TODO remove #postmigration, replaced by req.body.expense
       if (req.body.transaction) {
-        email = req.body.transaction.email;
-        paypalEmail = req.body.transaction.paypalEmail;
-        name = req.body.transaction.name;
+        ({ email } = req.body.transaction);
```

Comment:
didn't know you could do that. It treats all local vars as part of a single object? Are the parens on the outside needed?